### ISSUE TITLE
Vh s2 idea

### ISSUE THREAD

**Issue by jaykarsten** [Iskallia/Vault-public-S1#129]
I have an idea for vault hunters s2, maybe you could add proximity chat, bcause i've been reading comments and a few people said it was boring bcause there was no interaction, so maybe add proximity chat for more interaction?

**Comment by magicalex234** [comment]
I think you're in the wrong place. This forum is for direct issues with the code/pack itself, not really for suggestions.

**Comment by jaykarsten** [comment]
I was asked by a moderator in the discord to put it in issues so it will be
seen

**Comment by kotkowski** [comment]
1) It'd be better with coop vaults
2) Pretty sure proximity chat is verylaggy server-side without the use of outer software like mumble

**Comment by jaykarsten** [comment]
Fair point about the coop vaults, but yea, what i mean is using mumble or
any other software for prox chat, but if they are trading it will be easier
for them and more interaction for the viewers
